docs/README.md

```diff
@@ -2,41 +2,36 @@
 
 To get started, do the following:
 
-1. [Request an Echidna token](https://www.w3.org/Web/publications/register) for your spec (W3C Team Members and Chairs only!).
-1. Save the token as a "Secret" in the spec's repository's settings.
-1. Create a ".github/workflows/auto-publish.yml" file at the root of the spec's repository.
-   If you are using VS Code:
-   ```bash"
-   code .github/workflows/auto-publish.yml
-1. In to the "auto-publish.yml", copy, paste, and modify an one of the
-   [[[#examples]]] below that suits your needs. Most typical one:
+1. If you want to deploy to W3C /TR:
+   1. [Request an Echidna token][request-token] for your spec (W3C Team Members and Chairs only!).
+   1. [Save the token][save-token] as a "Secret" named `ECHIDNA_TOKEN` in the spec's repository's settings.
+1. Create a `.github/workflows/auto-publish.yml` file at the root of the spec's repository.
+1. In the `auto-publish.yml`, copy-paste and modify one of the [examples](examples.md) below that suits your needs. Most typical one:
 
    ```yml
-    # Inside .github/workflows/auto-publish.yml
-    name: Node CI
+   # Inside .github/workflows/auto-publish.yml
+   name: CI
```

Comment:
I think we should rename the name to something more obvious. I was scared to change it as I didn't know if it did anything.

```suggestion
   name: Automatic Publication 
```